.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
 permissions:
   contents: write
   pull-requests: write
+  packages: write
```

Comment:
Why is this needed?

Reply:
I was receiving a 403 when the tool was trying to deploy the artifacts to the GitHub package registry. After reproducing and testing this in a test repository, this option was the only one that fixed the issue.

Reply:
Interestingly, the deploy workflow does not need this and works fine. 🤔

Reply:
That was also my question, how come the deploy workflow works without this permission? 🤔